src/stylesheets/app.scss

```diff
@@ -1 +1,16 @@
+//base
+@import "base/colors";
 @import "base/typography";
+
+//functions
+@import "functions/colors";
+
+//partials
+@import "partials/behind-the-scenes";
+@import "partials/closing";
+@import "partials/concept";
+@import "partials/footer";
+@import "partials/header";
+@import "partials/key-factors";
+@import "partials/problem";
+@import "partials/question";
```

Comment:
If you wanna take a look at using <a href="https://www.npmjs.com/package/gulp-sass-glob">sass globbing</a>, this will clean this up and make it easier to have new partials automatically added.

Reply:
Thanks for the tip. I implemented  [node-sass-glob-importer](https://www.npmjs.com/package/node-sass-glob-importer) since it seemed a little easier to configure.